core/src/dird/dird_conf.cc

```diff
@@ -133,7 +133,7 @@ static ResourceItem dir_items[] = {
 #endif
   { "Subscriptions", CFG_TYPE_PINT32, ITEM(res_dir, subscriptions), 0, CFG_ITEM_DEFAULT, "0", "12.4.4-", NULL },
   { "MaximumConcurrentJobs", CFG_TYPE_PINT32, ITEM(res_dir, MaxConcurrentJobs), 0, CFG_ITEM_DEFAULT, "1", NULL, NULL },
-  { "MaximumConnections", CFG_TYPE_PINT32, ITEM(res_dir, MaxConnections), 0, CFG_ITEM_DEFAULT, "30", NULL, NULL },
+  { "MaximumConnections", CFG_TYPE_PINT32, ITEM(res_dir, MaxConnections), 0, CFG_ITEM_DEFAULT, "999999999999", NULL, NULL },
```

Comment:
Maybe we should not overpast the limit of the type UINT32_MAX or whatever the underlying CFG_TYPE_PINT32 is ?

Reply:
ok used a simpler enough value 100'000